lib/sign-up.js

```diff
@@ -23,6 +23,8 @@ function signUp (state, options) {
     return Promise.reject(new Error('SignUp with profile data not yet implemented. Please see https://github.com/hoodiehq/hoodie-account-client/issues/11.'))
   }
 
+  options.createdAt = new Date().toISOString()
```

Comment:
> Is the purpose of implementing createdAt to check how long a user has been using the app anonymously before signing up? hoodie/hoodie#282

Yes, exactly :)

> Looking at the difference in createdAt to the store signedUpAt, you'll render a different view?

Hoodie itself doesn’t do any UI rendering, but as the app developer you can treat users differently based on that information, like nudging people to finally sign up if they use the app anonymously since 2 weeks

> Or should I add the createdAt in [here](https://github.com/hoodiehq/hoodie-account-client/blob/master/utils/get-state.js#L44)?

Yes, that’s perfect

> if a User exists then it should already have the createdAt attached to the account

Yes, it will be stored in localStorage together with the `account.id`. It will be returned by [getAccount](https://github.com/hoodiehq/hoodie-account-client/blob/master/utils/get-state.js#L24) next time you load the page

> Sorry if my explanation is unclear. I'll try my best to further explain any part that sounds confusing.

Your explanation have been great :) I hope my comments make sense, please let me know if you have any further questions :)